src/Services/BaseService.php

```diff
@@ -33,6 +33,23 @@ public function __construct($table)
         $this->autoIncrements = self::getAutoIncrements($table);
     }
 
+    /**
+     * get
+     * Build SQL Query for Selecting Fields
+     *
+     * @param array $map
+     * @return array
+     */
+    public function get($map = null, $data = null){
```

Comment:
can't use get() here because facility service already has a protected get function.
In base classes that get extended best to use unique method names ie queryFields().

Reply:
Ohhh, Thanks I will update this in next commit 👍